views/js/runner/plugins/security/sectionPause.js

```diff
@@ -29,7 +29,7 @@ define([
 ], function(_, __, pluginFactory, mapHelper) {
     'use strict';
 
-    var pauseMessage = __('Awaiting for the proctor authorization.');
+    var pauseMessage = __('All students will begin the next section at the same time. Please relax quietly until the room supervisor starts the next section.');
```

Comment:
May I suggest "Please patiently await for your test supervisor to start the next section."

Why...
- Will all students actually begin at the same time? Isn't it a feature to begin test takers at different times?
- 'Patiently' and 'relax quietly' are synonyms... I guess I just prefer 'patiently.'
- Do all students await in the same room? I think 'test supervisor' is better than 'room supervisor.'